src/status_backend/config.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-backend.config)
```

Comment:
This file should be simplified with our nix tooling. But currently you need to add your own paths

Reply:
Nope, this is everything. I'll use `:clojure-defines`. Thank you!